src/runtime_src/core/tools/xbutil2/SubCmdClock.cpp

```diff
@@ -74,7 +85,7 @@ int subCmdClock(const std::vector<std::string> &_options, bool _help)
   }
 
   // Check to see if help was requested or no command was found
-  if (_help == true)  {
+  if (help == true)  {
```

Comment:
Is the bool_switch working? I tried the same implementation in xbmgmt, but it doesn't seem to work. It works under 2 scenarios:
1. change help=true in the beginning and this condition to check if help=false
2. or change --help to -h